src/core/secure/storage.ts

```diff
@@ -25,3 +25,11 @@ export const readEncrypted = async (key: string, hash: string) => {
         return Promise.reject(e);
     }
 };
+
+export const deleteWalletFromStorage = async (key: string) => {
```

Comment:
we should rename this, as it's a generic function, i would use `deleteEncrypted` or `deleteFromStorage`